### DISCUSSION TITLE
Software & Music (Nima)

### DISCUSSION THREAD

**Discussion by nsepehr** [WeAreMahsaAmini/Core#212] [Introductions]
Hello fellow community,

I'm a Musician & Software Engineer. 

- **Coding**: can code in JavaScript, Python, Go. Have a lot of knowledge on Front-End. Can do a lot in Backend. Not too strong in db setup, but can get simple things done well. 
- **Music**: I play Santur, flutes and can produce music with Ableton Live.

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163. 
please write a comment or let me know what you think about it. 
So many thanks*
